`make install` error with `--target=arm-apple-ios`

```shell
./configure --target=arm-apple-ios && make clean && make && make install
-- SNIP --
cleaning destination tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/bin
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/etc
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/share/man/man1
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libstd-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libregex-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/liblog-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libterm-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libserialize-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libfmt_macros-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libarena-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libsyntax-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libflate-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libgetopts-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librbml-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libgraphviz-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_llvm-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_back-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_borrowck-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_typeck-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_resolve-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_trans-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustc_driver-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/libtest-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/librustdoc-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/bin/rustdoc
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/share/man/man1/rustdoc.1
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/bin/rustc
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/share/man/man1/rustc.1
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/liblibc-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libstd-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libstd-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libflate-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libflate-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libarena-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libarena-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libterm-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libterm-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libserialize-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libserialize-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libgetopts-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libgetopts-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libcollections-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libtest-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libtest-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librand-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/liblog-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/liblog-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libregex-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libregex-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libgraphviz-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libgraphviz-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libcore-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librbml-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librbml-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/liballoc-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libunicode-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libsyntax-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_typeck-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_borrowck-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_resolve-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_driver-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_trans-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_back-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustc_llvm-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/librustdoc-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libfmt_macros-*.dylib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libmorestack.a
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/x86_64-apple-darwin/lib/libcompiler-rt.a
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/arm-apple-ios/lib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/arm-apple-ios/lib/liblibc-*.rlib
prepare: tmp/dist/rustc-1.0.0-dev-x86_64-apple-darwin-image/lib/rustlib/arm-apple-ios/lib/libstd-*.a
ls: x86_64-apple-darwin/stage2/lib/rustlib/arm-apple-ios/lib/libstd-*.a: No such file or directory
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make[1]: *** [prepare-target-arm-apple-ios-host-x86_64-apple-darwin-2-dir-x86_64-apple-darwin] Error 64
make: *** [install] Error 2
```

Full gist: https://gist.github.com/tamird/111eb6d0d0f5aa7ceda7

--- a/mk/cfg/aarch64-apple-ios.mk
+++ b/mk/cfg/aarch64-apple-ios.mk
@@ -11,7 +11,7 @@ AR_aarch64-apple-ios = $(shell xcrun -find -sdk iphoneos ar)
 endif
 CFG_LIB_NAME_aarch64-apple-ios = lib$(1).a
 CFG_LIB_GLOB_aarch64-apple-ios = lib$(1)-*.a
-CFG_LIB_SKIP_INSTALL_aarch64-apple-ios = 1 #lib$(1)-*.a
+CFG_INSTALL_ONLY_RLIB_aarch64-apple-ios = 1
 CFG_STATIC_LIB_NAME_aarch64-apple-ios=lib$(1).a
 CFG_LIB_DSYM_GLOB_aarch64-apple-ios = lib$(1)-*.a.dSYM
 CFG_CFLAGS_aarch64-apple-ios := $(CFG_IOS_SDK_FLAGS_aarch64-apple-ios)

--- a/mk/cfg/armv7-apple-ios.mk
+++ b/mk/cfg/armv7-apple-ios.mk
@@ -11,7 +11,7 @@ AR_armv7-apple-ios = $(shell xcrun -find -sdk iphoneos ar)
 endif
 CFG_LIB_NAME_armv7-apple-ios = lib$(1).a
 CFG_LIB_GLOB_armv7-apple-ios = lib$(1)-*.a
-CFG_LIB_SKIP_INSTALL_armv7-apple-ios = 1 #lib$(1)-*.a
+CFG_INSTALL_ONLY_RLIB_armv7-apple-ios = 1
 CFG_STATIC_LIB_NAME_armv7-apple-ios=lib$(1).a
 CFG_LIB_DSYM_GLOB_armv7-apple-ios = lib$(1)-*.a.dSYM
 CFG_JEMALLOC_CFLAGS_armv7-apple-ios := -arch armv7 -mfpu=vfp3 $(CFG_IOS_SDK_FLAGS_armv7-apple-ios)

--- a/mk/cfg/armv7s-apple-ios.mk
+++ b/mk/cfg/armv7s-apple-ios.mk
@@ -11,7 +11,7 @@ AR_armv7s-apple-ios = $(shell xcrun -find -sdk iphoneos ar)
 endif
 CFG_LIB_NAME_armv7s-apple-ios = lib$(1).a
 CFG_LIB_GLOB_armv7s-apple-ios = lib$(1)-*.a
-CFG_LIB_SKIP_INSTALL_armv7s-apple-ios = 1 #lib$(1)-*.a
+CFG_INSTALL_ONLY_RLIB_armv7s-apple-ios = 1
 CFG_STATIC_LIB_NAME_armv7s-apple-ios=lib$(1).a
 CFG_LIB_DSYM_GLOB_armv7s-apple-ios = lib$(1)-*.a.dSYM
 CFG_JEMALLOC_CFLAGS_armv7s-apple-ios := -arch armv7s -mfpu=vfp4 $(CFG_IOS_SDK_FLAGS_armv7s-apple-ios)

--- a/mk/cfg/i386-apple-ios.mk
+++ b/mk/cfg/i386-apple-ios.mk
@@ -11,6 +11,7 @@ AR_i386-apple-ios = $(shell xcrun -find -sdk iphonesimulator ar)
 endif
 CFG_LIB_NAME_i386-apple-ios = lib$(1).a
 CFG_LIB_GLOB_i386-apple-ios = lib$(1)-*.dylib
+CFG_INSTALL_ONLY_RLIB_i386-apple-ios = 1
 CFG_STATIC_LIB_NAME_i386-apple-ios=lib$(1).a
 CFG_LIB_DSYM_GLOB_i386-apple-ios = lib$(1)-*.dylib.dSYM
 CFG_GCCISH_CFLAGS_i386-apple-ios := -Wall -Werror -g -fPIC -m32 $(CFG_IOSSIM_FLAGS_i386-apple-ios)

--- a/mk/cfg/x86_64-apple-ios.mk
+++ b/mk/cfg/x86_64-apple-ios.mk
@@ -11,7 +11,7 @@ AR_x86_64-apple-ios = $(shell xcrun -find -sdk iphonesimulator ar)
 endif
 CFG_LIB_NAME_x86_64-apple-ios = lib$(1).a
 CFG_LIB_GLOB_x86_64-apple-ios = lib$(1)-*.a
-CFG_LIB_SKIP_INSTALL_x86_64-apple-ios = 1 #lib$(1)-*.a
+CFG_INSTALL_ONLY_RLIB_x86_64-apple-ios = 1
 CFG_STATIC_LIB_NAME_x86_64-apple-ios=lib$(1).a
 CFG_LIB_DSYM_GLOB_x86_64-apple-ios = lib$(1)-*.a.dSYM
 CFG_CFLAGS_x86_64-apple-ios := $(CFG_IOSSIM_FLAGS_x86_64-apple-ios)

--- a/mk/prepare.mk
+++ b/mk/prepare.mk
@@ -134,7 +134,7 @@ prepare-target-$(2)-host-$(3)-$(1)-$(4): prepare-maybe-clean-$(4) \
         $$(if $$(findstring $(3), $$(PREPARE_HOST)), \
           $$(call PREPARE_DIR,$$(PREPARE_WORKING_DEST_LIB_DIR)) \
           $$(foreach crate,$$(TARGET_CRATES), \
-	    $$(if $$(findstring 1, $$(ONLY_RLIB_$$(crate))),, \
+	    $$(if $$(or $$(findstring 1, $$(ONLY_RLIB_$$(crate))),$$(findstring 1,$$(CFG_INSTALL_ONLY_RLIB_$(2)))),, \
               $$(call PREPARE_LIB,$$(call CFG_LIB_GLOB_$(2),$$(crate)))) \
             $$(call PREPARE_LIB,$$(call CFG_RLIB_GLOB,$$(crate)))) \
           $$(if $$(findstring $(2),$$(CFG_HOST)), \

